docs/source/tutorials/describing_tasks.ipynb

```diff
@@ -1,20 +1,563 @@
 {
```

Comment:
<p>I am not sure we properly highlight the differences between <code>ranks</code> and <code>cores_per_ranks</code>.I tried to edit it but did not find (yet) a better way. We may want to collectively think about it.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2879/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I think the difference between <code>X</code> and <code>something_per_X</code> is intuitively clear, even if one does not exactly know what <code>X</code> is precisely (which I assume is what you are worried about?).  Don't you think so? </p>

Reply:
Only if `ranks` (X in your example) is properly described. The current text does not do that IMHO. About the intuitively clear, our users seem to disagree as they apparently stumbled over this.

Reply:
Right, that's what I meant, sorry if I wasn't clear.  We can lift the rank description from the docs and copy it to this place also.

Reply:
Done - added a similar description here.